docs/source/user_guide/installation/gnu-linux-mac.rst

```diff
@@ -14,16 +14,26 @@ To use PyBaMM, you must have Python 3.8, 3.9, 3.10, 3.11, or 3.12 installed.
 
    .. code:: bash
 
-      sudo apt update
-      sudo apt install python3
+      apt update \
+      apt install python3
```

Comment:
```suggestion
      sudo apt-get update \
      sudo apt-get install python3
```

this is what we've been using for our CI, I guess it makes sense? Also, removing `sudo` could be a bit misleading.

Also, most Debian-based distributions would not contain the latest and greatest Python distributions, could you add a sentence and a link to the Python official releases for those who wish to install using them?